### ISSUE TITLE
Service Mark Special character not read by screen reader on any setting

### ISSUE THREAD

**Issue by levyadams** [FreedomScientific/VFO-standards-support#438]
### Summary

> Brief description of the issue. Please list any specific steps.
> JAWS reads most other special legal characters like copyright and trademark. Service mark is important for companies who do not offer products, but rather services. Service mark is a legally binding character that needs to be read out.
> Example:
> Screen read the ℠ character on any setting for punctuation 
> JAWS does not read the service mark on any setting.

### Expected result

> What did you expect to happen?

Expect to hear, "service mark" just like spoken for trademark and copyright.
### Actual result
No sound from screen reader.
> What actually did happen?
Nothing
### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

Use a screen reader on this paragraph and notice the ℠ isn't read out
#### JAWS version and build number

17.0.2007
 
#### Operating System and version

windows 10 enterprise

#### Browser and version:

Chrome 85.0.4183.83

**Comment by JAWS-test** [comment]
This problem also occurs with JAWS 2020. It affects not only the Service Mark character, but many special characters

**Comment by stevefaulkner** [comment]
@levyadams I am following up on this internally.

**Comment by levyadams** [comment]
> @levyadams I am following up on this internally.

If it helps at all, I raised this same issue with the open-source Screen Reader Non Visual Desktop Assistant (NVDA). We identified the file the special characters verbosity was located and added, "ServiceMark" to the "some" punctuation setting.

Here is the PR that also links to the issue. Not sure if this helps the dev at all but the fix seems simple.
https://github.com/nvaccess/nvda/pull/11559

Is there any way I can track this issue further? What is the usual deployment of software updates for JAWS? I currently have a large client who is worried about their 508 compliance with this issue and would love to be able give them some sort of even loosely based time frame of when this change will go into effect. They strictly use your software for their compliance testing and only default to NVA when they cannot get a license or don't have proper equipment for JAWS testing.

Thank you so much. Have a great week! :D
